docs/site.md

```diff
@@ -6,13 +6,18 @@ Parts of the RK² web site are mined from game data; most notably the relic draw
 
```

Comment:
Are your changes rebased on mine?  I'm not sure I want to commit all of my stuff just yet, so maybe keeping yours separate when merging is better?

Reply:
Yup, I can remove the merge from the branch, the reason I have to merge it in is because the DASB parsing without your changes are breaking RkSquared